source/Ocl/Converters/EnumAttributeOclConverter.cs

```diff
@@ -33,7 +33,7 @@ public OclDocument ToDocument(OclConversionContext context, object obj)
 
         public IEnumerable<IOclElement> ToElements(OclConversionContext context, PropertyInfo? propertyInfo, object obj)
         {
-            var isDefault = Activator.CreateInstance(obj.GetType()).Equals(obj);
+            var isDefault = Equals(Activator.CreateInstance(obj.GetType()), obj);
```

Comment:
.NET 6 was complaining about calling `.Equals()` on the potential null returned by `CreateInstance()`